scrapy/utils/template.py

```diff
@@ -10,7 +10,7 @@ def render_templatefile(path, **kwargs):
 
     content = string.Template(raw).substitute(**kwargs)
 
-    with open(path.rstrip('.tmpl'), 'wb') as file:
+    with open(re.sub(r'\.tmpl$', '', path), 'wb') as file:
```

Comment:
I prefer to use simple string functions instead of regex whenever I can, my suggestion is to use `path.rpartition('.')[0]` if this is about striping the template extension only.
In any case it looks good to me.